introduction.md

```diff
@@ -0,0 +1,3 @@
+Hey Boston College. Your school is a great school but I choode not to come there because it is too good for me.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.